pipelines/deployer/deployer.yaml

```diff
@@ -731,10 +731,47 @@ jobs:
                   echo "done"
               fi
           fi
+  - task: generate-terraform-var-overrides
```

Comment:
Have you confirmed that this outputs an empty map `{}` when there are no autoscaling groups (as in when this is a brand new cluster)? .... looks like it should be good so long as `describe-auto-scaling-groups` doesn't do something weird.

Reply:
yep